src/poprox_recommender/default.py

```diff
@@ -54,20 +52,19 @@ def parse_row(token_mapping, row):
     )
 
 
-# check the input type of articles 
-def is_list_of_dicts(articles): 
+# check the input type of articles
+def is_list_of_dicts(articles):
```

Comment:
With the changes to `transform_article_features` below, I think this function can now be removed